### DISCUSSION TITLE
How to put DNS provider API keys in keyvault instead of env variable

### DISCUSSION THREAD

**Discussion by BarryBahrami** [polymind-inc/acmebot#790] [General]
Is it possible to put the GoDaddy DNS provider API key and secret into the keyvault rather than saving them as an environment variable?

than you in advance

Barry

**Comment by shibayan** [comment]
This is possible using the Key Vault Reference.

- https://learn.microsoft.com/en-us/azure/app-service/app-service-key-vault-references?tabs=azure-cli
